calico/reference/felix/configuration.md

```diff
@@ -69,7 +69,7 @@ The full list of parameters which can be set is as follows.
 | `RemoveExternalRoutes`            | `FELIX_REMOVEEXTERNALROUTES`            | Whether or not to remove device routes that have not been programmed by Felix. Disabling this will allow external applications to also add device routes. [Default: `true`] | bool |
 | `ReportingIntervalSecs`           | `FELIX_REPORTINGINTERVALSECS`           | Interval at which Felix reports its status into the datastore or `0` to disable. Must be non-zero in OpenStack deployments. [Default: `30`] | int |
 | `ReportingTTLSecs`                | `FELIX_REPORTINGTTLSECS`                | Time-to-live setting for process-wide status reports. [Default: `90`] | int |
-| `RouteTableRange`                 | `FELIX_ROUTETABLERANGE`                 | Calico programs additional Linux route tables for various purposes.  `RouteTableRange` specifies the indices of the route tables that Calico should use. [Default: `1-250`] | `<min>-<max>` |
+| `RouteTableRanges`                 | `FELIX_ROUTETABLERANGES`                 | Calico programs additional Linux route tables for various purposes. `RouteTableRanges` designates a set of table ranges that Calico is permitted to use. [Default: `1-250`] | `<min>-<max>,<min>-<max>,...` |
```

Comment:
Should the range here match the below value of `1-250,256-10000`?

Reply:
yes it should, good catch!

Reply:
or alternatively, I could change the other one to be 1-250, to maintain backward compat :thinking:

Reply:
Yeah, you're right, probably something we need to get @fasaxc 's perspective on.

Reply:
final approach is to specify `1-10000` and let felix ignore reserved tables